view/adminhtml/web/template/view/certificates-grid.html

```diff
@@ -0,0 +1,54 @@
+<!-- ko if: shouldShowWarning -->
+<p class="message">
+    Please note that you can only view certificates for a customer from the website/store that customer
+    is associated to. This means if you have 2 different Avalara credentials for 2 different stores and one customer
+    uploads certificates to both, only certificates uploaded through the store that customer is associated to will
+    be visible here.
+</p>
+<!-- /ko -->
+<div class="admin__data-grid-wrap admin__data-grid-wrap-static">
+    <table class="data-grid">
+        <thead>
+            <tr>
+                <th class="data-grid-th">State</th>
+                <th class="data-grid-th">Signed Date</th>
+                <th class="data-grid-th">Expiration Date</th>
+                <th class="data-grid-th">Status</th>
+                <th class="data-grid-th" colspan="3">&nbsp;</th>
+            </tr>
+        </thead>
+
+        <tbody>
+            <!-- ko if: certificates.length > 0 -->
+                <!-- ko foreach: certificates -->
+                    <tr data-role="row">
+                        <td data-bind="text: exposure_zone.name"></td>
+                        <td data-bind="text: signed_date"></td>
+                        <td data-bind="text: expiration_date"></td>
+                        <td data-bind="text: valid ? 'Yes' : 'No'"></td>
```

Comment:
Not sure the best way to handle it, but you need to make sure that "Yes" and "No" are translatable